src/components/dashboard/ClaimButton.js

```diff
@@ -51,10 +54,21 @@ const ButtonAmountToClaim = ({ showLabelOnly = false, entitlement, isCitizen, st
 )
 
 export const ButtonCountdown = ({ styles, nextClaim }) => (
-  <View style={styles.countdownContainer}>
-    <Text style={styles.extraInfoCountdownTitle} fontWeight="bold">
-      Your next daily claim:
-    </Text>
+  <View style={isSmallDev ? styles.countdownContainerSmallDev : styles.countdownContainer}>
+    {isSmallDev ? (
```

Comment:
are we couldn't just fix up the text width  and allow it to be wrapped onto new line if it doesn't fits ?

Reply:
@serdiukov-o-nordwhale I think the design requires these lines to be in 2 lines on mobile. By the way, this component is not used anymore since the new design was implemented - as it is the part of the old design.

Reply:
try to set numberOfLines={2}
if text won't fit width it should be wrapped onto new line

Reply:
I tried to refactor it with numberOfLines property but it is not so easy as we think/want - I spend some time on it and I think that it's not so critical to spend a much more.

Reply:
ok